tensorflow_data_validation/coders/decoder_test_util.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utils for decoder tests."""
```

Comment:
Instead of adding a new file for this method, can you add it to [test_util.py](https://github.com/tensorflow/data-validation/blob/master/tensorflow_data_validation/utils/test_util.py)?